filebeat/channel/outlet.go

```diff
@@ -90,6 +89,6 @@ func (o *Outlet) OnEventSignal(event *input.Data) bool {
 	}
 }
 
-func (o *Outlet) Copy() prospector.Outlet {
+func (o *Outlet) Copy() Outleter {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Outlet.Copy should have comment or be unexported